src/clr-angular/utils/drag-and-drop/interfaces/draggable.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+export interface Draggable {
```

Comment:
Could you please comment what the interface does as well as what the properties are responsible for?

Reply:
Will do.